app/src/main/kotlin/com/mapbox/vision/teaser/utils/PermissionsUtils.kt

```diff
@@ -30,12 +30,12 @@ object PermissionsUtils {
     }
 
     fun arePermissionsGranted(activity: MainActivity, requestCode: Int) =
-        requestCode == PERMISSIONS_REQUEST_CODE
-        && getNotGrantedPermissions(activity).none { permission ->
+        requestCode == PERMISSIONS_REQUEST_CODE &&
+                getNotGrantedPermissions(activity).none { permission ->
```

Comment:
But why? I like conditions formatted without indents very much, @yuryybk @mapbox/vision-android lets vote

Reply:
I mostly used and saw shifted conditions at the next line (not aligned with previous).
But don't have strict mind regarding any style

Reply:
It's what `ktlint` advised actually.
AFAIR it's not advised by Kotlin itself to start a line with such symbols.

Reply:
Hmm. 
I like it since it looks logical that if parts are equal by their order - they should be aligned as well. 
Contrary, if next line is logically a subordinate - it's clear to have continuation indent there. 

In this case I'd rather have

```
a &&
b &&
c
```
(&& might go in the end of the current line or in the start of the following - not reallty important)

then 
```
a &&
    b && 
    c
```

Also, second option is kind of assymetric, since continuation indent is only added to the second line, third line is not indented relatively to the second one

Reply:
@yunikkk agree here - fixed indent, PTAL

Reply:
Cool, thanks

Reply:
But please get rid of the indent above as well

Reply:
It's too late...